backtrace-library/src/main/java/backtraceio/library/common/RequestHelper.java

```diff
@@ -0,0 +1,53 @@
+package backtraceio.library.common;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+import backtraceio.library.logger.BacktraceLogger;
+
+public class RequestHelper {
+    private static final transient String LOG_TAG = RequestHelper.class.getSimpleName();
+
+    private static final String CRLF = "\r\n";
+
+    /**
+     * Get Content-Type of request
+     *
+     * @return string with content type and information about boundary
+     */
+    public static String getContentType() {
+        return "application/json";
+    }
+
+    /**
+     * Write JSON string to output data steam
+     *
+     * @param outputStream output data stream
+     * @param json         JSON string with BacktraceData object
+     * @throws IOException
+     */
+    public static void addJson(OutputStream outputStream, String json) throws IOException {
+        if (json == null || json.isEmpty() || outputStream == null) {
+            BacktraceLogger.w(LOG_TAG, "JSON is null/empty or output stream is null");
+            return;
+        }
+
+        byte[] bytes = json.getBytes("utf-8");
```

Comment:
Extract constant. Why UTF8? Are there cases it's not UTF8? Should the charset be encoding in the application/json

> Content-type: application/json; charset=utf-8

Reply:
We want to keep encoding the same everywhere. In all our libraries we're usnig utf-8. 

extracting constant makes sense however it's not that important from the pull request perspective.

Reply:
Again I just followed @BartoszLitwiniuk style:
https://github.com/backtrace-labs/backtrace-android/blob/dcf0eb97675d693c5aff0f733d407197e9dad168/backtrace-library/src/main/java/backtraceio/library/common/MultiFormRequestHelper.java#L63
But we can extract this constant since we do it for `CRLF`, `BOUNDARY`, `TWO_HYPHENS`, already